tests/lib/ocpp/v201/CMakeLists.txt

```diff
@@ -7,5 +7,10 @@ target_sources(libocpp_unit_tests PRIVATE
         test_notify_report_requests_splitter.cpp
         test_ocsp_updater.cpp
         test_component_state_manager.cpp
+        test_composite_schedule.cpp
         test_device_model.cpp
-        test_smart_charging_handler.cpp)
+        test_smart_charging_handler.cpp
+        smart_charging_test_utils.hpp
+        )
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/json DESTINATION /tmp/EVerest/libocpp/v201)
```

Comment:
We probably want to install this somewhere in the build directory or the install prefix instead of hardcoding to `/tmp`.

Reply:
As discussed before, I have not found a good method to importing relative to the project. This is a great refactoring point for after the merge.